packages/react-instantsearch-hooks-server/src/__tests__/getServerState.test.tsx

```diff
@@ -362,19 +378,58 @@ describe('getServerState', () => {
       },
     });
   });
-});
 
-function SearchBox() {
-  useSearchBox();
-  return null;
-}
-
-function Hits() {
-  useHits();
-  return null;
-}
+  test('returns HTML from server state', async () => {
+    const searchClient = createSearchClient({});
+    const { App } = createTestEnvironment({ searchClient });
 
-function RefinementList(props: UseRefinementListProps) {
-  useRefinementList(props);
-  return null;
-}
+    const serverState = await getServerState(<App />);
+    const html = renderToString(<App serverState={serverState} />);
+
+    expect(html).toMatchInlineSnapshot(`
+      <div class="ais-RefinementList ais-RefinementList--noRefinement">
+        <ul class="ais-RefinementList-list">
+        </ul>
+      </div>
+      <div class="ais-SearchBox">
+        <form action
+              class="ais-SearchBox-form"
+              novalidate
+        >
+          <input class="ais-SearchBox-input"
+                 type="search"
+                 value="iphone"
+          >
+        </form>
+      </div>
+      <h2>
+        instant_search
+      </h2>
+      <div class="ais-Hits ais-Hits--empty">
+        <ol class="ais-Hits-list">
+        </ol>
```

Comment:
isn't this meant to have li's? or is the search client returning empty hits?

Reply:
It returns no hits, I added some.